Sources/Views/MessageInputBar.swift

```diff
@@ -520,7 +520,6 @@ open class MessageInputBar: UIView {
                     $0.parentStackViewPosition = position
                     leftStackView.addArrangedSubview($0)
                 }
-                guard superview != nil else { return }
```

Comment:
Thanks for your `PR`, It's required, because we cannot call `layoutIfNeeded()` when view has not been added to the `superview`.